web/packages/teleport/src/lib/tdp/client.ts

```diff
@@ -756,9 +760,9 @@ export default class Client extends EventEmitterMfaSender {
     this.send(this.codec.encodeSharedDirectoryTruncateResponse(response));
   }
 
-  resize(spec: ClientScreenSpec) {
+  resize = (spec: ClientScreenSpec) => {
     this.sendClientScreenSpec(spec);
-  }
+  };
 
```

Comment:
changing to a fat arrow function makes me think that `this` is now used correctly, but how was it working before without the lexical binding?

Reply:
It was wrapped in an arrow function:
```ts
  const onResize = (e: { height: number; width: number }) => {
     tdpClient.resize(e);
   };
```
I removed it and called the function directly.